### ISSUE TITLE
Media keys not working in Pocket Casts

### ISSUE THREAD

**Issue by bdutro** [tiliado/nuvolaruntime#659]
### Basic information

- Your Linux distribution and its version: Arch Linux, kernel version 5.9.1-arch1-1
   

- Your desktop environment: i3-gaps 4.18.3
   

 - Nuvola Runtime version: 4.17.25 (25-g72fcd17)
   

 - The web app you use and its version: Pocket Casts 1.3.0 (0-gcba8da7)
   

### Describe the bug

Media keys stopped working to play/pause and skip forwards/backwards. I also tried manually triggering these events through MPRIS using `playerctl` and got the same results.

**Steps to reproduce the behavior**

1. Open Pocket Casts
2. Use play/pause media key - nothing happens
3. Manually start playback in Pocket Casts
4. Try skipping forwards/backwards with media key - nothing happens

**Expected behavior**

All media key shortcuts should work.

**Debugging log**

Attach the debug log here: 
[debug.log](https://github.com/tiliado/nuvolaruntime/files/5423906/debug.log)

**Additional context**

The last time I updated Nuvola and the Pocket Casts app, the play/pause shortcut stopped working, but the skip forward/backwards shortcuts still worked. Now none of them work.

These lines in the debug log stuck out at me:
```
Runner: [WARNING Δ09s918625us DioriteGtk] Action.vala:69: Cannot activate action 'toggle-play', because it is disabled.
...
Runner: [INFO Δ000930us CefGtk] WebView.vala:106: Console: file:///app/share/nuvolaruntime/js/main.js:105: JS API: Action activated: prev-song
Runner: [DEBUG Δ000140us DioriteGtk] Actions.vala:322: Action activated: playback/win.prev-song(null)
Runner: [WARNING Δ666131us Nuvola] Binding.vala:86: Call Web Worker sync: Nuvola.actions.emit
Runner: [INFO Δ000546us CefGtk] WebView.vala:106: Console: file:///app/share/nuvolaruntime/js/main.js:105: JS API: Action activated: next-song
Runner: [DEBUG Δ000066us DioriteGtk] Actions.vala:322: Action activated: playback/win.next-song(null)
```

**Comment by fenryxo** [comment]
Hello @bdutro. Thanks for your bug report.

**Comment by fenryxo** [comment]
Fixed and a fresh flatpak published (`1.3.1 (1-g744ddeb)`). Volume management was broken too. Thanks for reporting the issue :-)
